orion/controllers/programs/programs_test.go

```diff
@@ -11,9 +13,55 @@ func TestExample(t *testing.T) {
   }
 }
 
-func TestCheckProgram(t *testing.T) {
-  isValid := CheckValidProgram()
-  if (isValid == false) {
-    t.Errorf("Check was incorrect, got: %t, expected: %t.", isValid, false)
-  }
+func Rand(a, b int) uint {
+	return uint(rand.Int() % (b - a) + a)
 }
+
+func TestCheckProgram(t *testing.T) {
+	// success examples
+	var s string
+	for i := 0; i < 1000; i++ {
```

Comment:
Good thought but I think a random stress test like this is overkill for our problem at hand. We are checking for correctness here so we should keep tests granular and explicit. So what you have below is good enough.

We can think of stress testing once we start integrating http requests or sql querying